en/docs/integrations/build-your-first-integration.md

```diff
@@ -0,0 +1,59 @@
+# Build Your First Integration
+Choreo allows you to efficiently build integrations by connecting APIs, events, and streams. Choreo also provides a comprehensive set of prebuilt integration templates that you can select and use depending on your requirement.
+
+This quick start guide walks you through the steps to quickly configure and try out the **GitHub New Issue to Google Sheets Row** integration in Choreo.
+
+## Prerequisites:
+- A GitHub account to connect to in the integration configuration.
+- A Google account to connect to in the integration configuration.
+- A blank spreadsheet named `Sample GitHub Integration` created in the Google account. This is the spreadsheet name to use in the integration configuration. 
+  The spreadsheet will have a default worksheet named `Sheet1`. This is the worksheet name to use in the integration configuration.
+
+## Step 1: Configure and start the integration
+Follow the procedure below to configure and start the **GitHub New Issue to Google Sheets Row** integration:
+
+1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
+2. Sign in using either your GitHub or Google credentials.
+3. Go to the **Integrations** card and click **Get Started**. Now you are on the **Integrations** page, where you can start with a prebuilt integration or create your own.
+4. Go to the **GitHub New Issue to Google Sheets Row** card and click **Use This**.
+5. In the **Configurations** panel, fill in the details as follows to configure the integration:
+    1. Click **Connect to GitHub**.
+    2. In the authorization request page that opens, click **Authorize wso2** to allow WSO2 to access your GitHub account. 
+    3. In the GitHub access confirmation page that opens, enter your GitHub password and click **Confirm password**.
+    4. In the **GitHub Repository URL** field, select the repository for which you want to track new issues.
+    5. Click **Connect to Google Sheets**, and then click **Proceed**.'
+    6. Select the Google account you want to use in the integration.
+    7. In the access request page that opens, click **Allow** to proceed with the selected Google account.
+    8. In the **Spreadsheet Name** field, select `Sample GitHub Integration`.
+    9 . In the **Worksheet Name** field, enter `Sheet1`.
```

Comment:
It seems the number `9` is not properly added.

Reply:
Fixed.